src/com/epam/course/hometask3/Book.java

```diff
@@ -0,0 +1,90 @@
+package com.epam.course.hometask3;
+
+public class Book {
+    protected String name;
+
+
+    public Book (String name) {
+        this.name = name;
+    }
+
+    private String isbn;
+    private String author;
+    private String publisher;
```

Comment:
Same with publisher,  set this parameter type to the Publisher class you created